src/index.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter } from 'react-router-dom';
+import { ApolloProvider } from 'react-apollo';
+import { ApolloClient } from 'apollo-client';
+import { createHttpLink } from 'apollo-link-http';
+import { InMemoryCache } from 'apollo-cache-inmemory';
+
+import App from 'components/App';
+
+const httpLink = createHttpLink({
+  uri: 'http://localhost:4000'
```

Comment:
i think you should create uri for difference env. Eg:
let uri = 'http://localhost:4000'
if(process.env.NODE_ENV === 'production') {
   uri = 'uri for prod'
}
const httpLink = createHttpLink({uri})